tests/test_wmb_download.py

```diff
@@ -0,0 +1,5 @@
+from crmprtd.wmb_dir.download import logging_setup
+
+def test_logging_setup():
+    log = logging_setup('mof.log', 'test@mail.com', 'INFO')
+    assert log != None
```

Comment:
Are there any stronger guarantees that you can make about the return value of ``logging_setup()``? What type is it? What information does it need to contain to be functional for the rest of the program?